scipy/signal/tests/test_signaltools.py

```diff
@@ -1124,30 +1115,6 @@ def test_none(self):
         a.strides = 16
         assert_(signal.medfilt(a, 1) == 5.)
 
-    def test_refcounting(self):
```

Comment:
Object array test hopefully safe to remove

Reply:
This should be fine - and we should remove all tests that explicitly check the refcount (I'm not sure if we have any others), because with changes in CPython those tests can no longer be correct.